src/components/_internal/sw-label.vue

```diff
@@ -109,10 +109,10 @@ export default Vue.extend({
 
   computed: {
     labelClasses(): (string | {
-        [x: string]: boolean;
-        'sw-label--dismissable': boolean;
-        'sw-label--ghost': boolean;
-        'sw-label--caps': boolean;
+      [x: string]: boolean;
+      'sw-label--dismissable': boolean;
+      'sw-label--ghost': boolean;
+      'sw-label--caps': boolean;
     })[] {
```

Comment:
This was from the Vue2 time. It was necessary to type any computed value otherwise it couldn't figure out the types. I removed them because in Vue3 it can understand the types better